Integrations/integration-TCell.yml

```diff
@@ -0,0 +1,635 @@
+  required: true
+- display: Enter API Key (Full Access)
+  name: api_key
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: ""
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |+
+    var apiKey = params.api_key;
+    var server = params.server;
+    var proxy = params.proxy;
+    var insecure = params.insecure;
+
+    if (server[server.length - 1] === '/') {
+        server = server.substring(0, server.length - 1);
+    }
+
+    var urlDictionary = {
+            'tcell-list-apps': 'apps',
+            'tcell-get-app': 'apps/%APP_ID%',
+            'tcell-list-agents': 'apps/%APP_ID%/agents',
+            'tcell-agent-data': 'apps/%APP_ID%/agents/%AGENT_ID%',
+            'tcell-app-routes': 'apps/%APP_ID%/routes',
+            'tcell-app-route-data': 'apps/%APP_ID%/routes/%ROUTE_ID%',
+            'tcell-app-inline-scripts': 'apps/%APP_ID%/inline_scripts',
+            'tcell-app-inline-script-data':'apps/%APP_ID%/inline_scripts/%SCRIPT_ID%',
+            'tcell-app-packages':'apps/%APP_ID%/packages',
+            'tcell-app-package-data':'apps/%APP_ID%/packages/%PACKAGE_ID%',
+            'tcell-app-source-events':'apps/%APP_ID%/sources/%SOURCE%/events',
+            'tcell-app-events-table':'apps/%APP_ID%/sources/%SOURCE%/events/table',
+            'tcell-sources-event-data':'apps/%APP_ID%/sources/%SOURCE%/events/%EVENT_ID%',
+            'tcell-app-configs':'apps/%APP_ID%/configs',
+            'tcell-app-config-latest':'apps/%APP_ID%/configs/latest',
+            'tcell-app-config-data':'apps/%APP_ID%/configs/%CONFIG_ID%',
+            'tcell-app-push-config':'apps/%APP_ID%/configs'
+
+    };
+
+    var methodDictionary = {
+            'tcell-list-apps': 'GET',
+            'tcell-get-app': 'GET',
+            'tcell-list-agents': 'GET',
+            'tcell-agent-data': 'GET',
+            'tcell-app-routes': 'GET',
+            'tcell-app-route-data': 'GET',
+            'tcell-app-inline-scripts': 'GET',
+            'tcell-app-inline-script-data':'GET',
+            'tcell-app-packages':'GET',
+            'tcell-app-package-data':'GET',
+            'tcell-app-source-events':'GET',
+            'tcell-app-events-table':'GET',
+            'tcell-sources-event-data':'GET',
+            'tcell-app-configs':'GET',
+            'tcell-app-config-latest':'GET',
+            'tcell-app-config-data':'GET',
+            'tcell-app-push-config':'POST'
+
+    }
+
+
+    var tcellListApps = function() {
+        var respBody = sendRequest(methodDictionary['tcell-list-apps'], urlDictionary['tcell-list-apps'],'', '');
+        return {
+            Type: entryTypes.note,
+            Contents: respBody,
+            ContentsFormat: formats.json,
+            HumanReadable: tableToMarkdown('TCell Apps', respBody.apps),
+            EntryContext: {'tcell_apps':respBody.apps}
```

Comment:
we usually manipulate the data to extract the main things out of it and not pushing all the to context